mindsdb/integrations/handlers/financial_modeling_prep_handler/financial_modeling_handler.py

```diff
@@ -0,0 +1,61 @@
+import pandas as pd
+from typing import Dict
+from urllib.parse import urlencode
+from mindsdb.integrations.libs.api_handler import APIHandler
+from mindsdb.integrations.libs.response import (
+    HandlerStatusResponse as StatusResponse,
+    HandlerResponse as Response,
+    RESPONSE_TYPE
+)
+from mindsdb.integrations.handlers.financial_modeling_prep_handler.financial_modeling_tables import HistoricalPriceTable
+
+from urllib.request import urlopen
+from mindsdb.utilities import log
+import json
+import requests
+_FINANCIAL_MODELING_URL = 'https://financialmodelingprep.com/api/v3/'
+
+logger = log.getLogger(__name__)
+
+class FinancialModelingHandler(APIHandler):
```

Comment:
I believe you will need to add a `native_query()` method to your handler as well.

Reply:
I believe the native_query method is used when the API can take in raw SQL queries, however the financial modeling prep API doesn't. What should the native_query() method consist of in this case?

Reply:
Yes, you are correct! Sorry, I was mistaken.